starky/src/recursive_verifier.rs

```diff
@@ -51,10 +47,21 @@ pub fn verify_stark_proof_circuit<
     let max_degree_bits_to_support = proof_with_pis.proof.recover_degree_bits(inner_config);
 
     let mut challenger = RecursiveChallenger::<F, C::Hasher, D>::new(builder);
+    let degree_bits_target = proof_with_pis.proof.degree_bits;
```

Comment:
minor: probably no need to extract here, as the `get_challenges` method below take a proof as `&self`